From aab9ab2a584222482eea0cfd40696443b2ef1a33 Mon Sep 17 00:00:00 2001
From: Jeffrey Ladouceur <jeffrey.ladouceur@freescale.com>
Date: Wed, 14 Nov 2012 00:19:03 +0000
Subject: [PATCH 144/227] pme: Add pattern matcher to device tree

Add the Pattern Matcher accelerator to the T4240 device tree.

Signed-off-by: Jeffrey Ladouceur <Jeffrey.Ladouceur@freescale.com>
Signed-off-by: Andy Fleming <afleming@freescale.com>
[Kevin: Original patch taken from fsl sdk 1.3.1
QorIQ-SDK-V1.3.1-SOURCE-20121220-yocto.iso.]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/powerpc/boot/dts/fsl/t4240si-post.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/powerpc/boot/dts/fsl/t4240si-post.dtsi b/arch/powerpc/boot/dts/fsl/t4240si-post.dtsi
index 6d24164..c1b9275 100644
--- a/arch/powerpc/boot/dts/fsl/t4240si-post.dtsi
+++ b/arch/powerpc/boot/dts/fsl/t4240si-post.dtsi
@@ -306,6 +306,7 @@
 /include/ "qoriq-sata2-0.dtsi"
 /include/ "qoriq-sata2-1.dtsi"
 /include/ "qoriq-dce-0.dtsi"
+/include/ "qoriq-pme-0.dtsi"
 
 /include/ "qoriq-fman3-0.dtsi"
 /include/ "qoriq-fman3-0-1g-0.dtsi"
-- 
1.7.9.7

